src/main/java/org/folio/circulation/rules/AppliedRuleConditionsEntity.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.circulation.rules;
+
+public class AppliedRuleConditionsEntity {
```

Comment:
Why does this class name have the suffix `entity`?  What information does it provide over the name without it?

Reply:
renamed